inc/class-statify-frontend.php

```diff
@@ -469,7 +478,7 @@ private static function make_amp_config() {
 				'action'           => 'statify_track',
 				'_ajax_nonce'      => wp_create_nonce( 'statify_track' ),
 				'statify_referrer' => '${documentReferrer}',
-				'statify_target'   => '${canonicalPath}amp/',
+				'statify_target'   => '${canonicalPath}?amp',
```

Comment:
Same as above, why not use `add_query_arg`?

Reply:
What's the benefit?
`${canonicalPath}` is passed as is and will be replaced in the frontend logic by Javascript. `add_query_arg('amp', '${canonicalPath}')` will always return `${canonicalPath}?amp`.

Reply:
You are right, I was not aware of the frontend logic!